core/pom.xml

```diff
@@ -83,6 +83,7 @@
                 <buildver>311</buildver>
                 <spark.version>${spark311.version}</spark.version>
                 <delta.core.version>${delta10x.version}</delta.core.version>
+                <hadoop.version>3.2.0</hadoop.version>
```

Comment:
hopefully this doesn't matter to much because users could be using any version of spark that they could have compiled themselves against any verson of hadoop.  It should work most of the time, especially if they download the spark tgz.  If it does we may need to start bundling things

Reply:
That's right.
I changed it so that Unit-tests by default would use the same hadoop-version as the Spark-release. So, tests will be more accurate.
Before that change, we used to build against the "hadoop-version-update" which is not used by the actual spark-release.